Three/include/CGAL/Three/Scene_interface.h

```diff
@@ -144,11 +144,22 @@ class Scene_interface {
   //! Updates the information about `item` in the
   //! Geometric Objects list and redraws the scene.
   virtual void itemChanged(CGAL::Three::Scene_item* item) = 0;
+  //!
+  //! \brief Updates all the items in the SceneView.
+  //!
+  virtual void allItemsChanged() = 0;
```

Comment:
Each time you add a virtual method, anywhere, that is not the last one, then you break the ABI compatibility.

You should always add virtual functions after the other ones.